api/src/main/resources/application.properties

```diff
@@ -0,0 +1,34 @@
+#Keycloak/OAuth properties
+security.oauth2.resource.jwt.key-value=${KEYCLOAK_PUBLIC_KEY}
+
+#logging Properties
+logging.level.org.springframework.security=${SPRING_SECURITY_LOG_LEVEL}
+logging.level.org.springframework.web=${SPRING_WEB_LOG_LEVEL}
+logging.level.ca.bc.gov.educ.api.sld=${APP_LOG_LEVEL}
+logging.level.org.springframework.boot.autoconfigure.logging=${SPRING_BOOT_AUTOCONFIG_LOG_LEVEL}
+spring.mvc.log-request-details=${SPRING_SHOW_REQUEST_DETAILS}
+logging.pattern.console=%d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+
+#DB Properties
+spring.datasource.url=${JDBC_URL}
+spring.datasource.username=${ORACLE_USERNAME}
+spring.datasource.password=${ORACLE_PASSWORD}
+spring.jpa.database-platform=org.hibernate.dialect.Oracle12cDialect
+spring.jpa.hibernate.ddl-auto=none
+#So that unexpected request body parameters cause error
+spring.jackson.deserialization.fail-on-unknown-properties=true
+
+management.endpoint.metrics.enabled=true
+management.endpoints.web.exposure.include=*
+management.endpoint.prometheus.enabled=true
+management.metrics.export.prometheus.enabled=true
+spring.datasource.hikari.max-lifetime=120000
+spring.jmx.enabled=false
```

Comment:
please follow this application.properties and update https://github.com/bcgov/EDUC-PEN-TRAX-API/blob/master/api/src/main/resources/application.properties